### DISCUSSION TITLE
links to inphared files 1Jun2024 dead

### DISCUSSION THREAD

**Discussion by ColinTHB** [RyanCook94/inphared#27] [General]
To whom it may concern,

I am getting a 503 Service Unavailable message when downloading the above.

Many thanks,

Colin B (9/7/24)

**Comment by RyanCook94** [comment]
Hi Colin,

We currently house the files on an s3 bucket hosted by CLIMB-Big-Data but, unfortunately, CLIMB is currently down! I think they had a serious technical issue in of their data centres. Hopefully it will be back up soon.

In the mean time, if there's any files you need- I'm happy to email them to you if you drop me a line at ryan.cook@quadram.ac.uk

All the best,
Ryan
